### PR TITLE
MGMT-5283 Support IPv6 configurations for using pytest

### PR DESCRIPTION
**Pull request by eliorerz**

### CODE REVIEW

**Comment by eliorerz** [comment]
/hold
Waiting for https://github.com/openshift/release/pull/19297 to be merge

File: discovery-infra/tests/base_test.py [hunk 1]
```diff
@@ -109,6 +115,18 @@ def get_cluster_func(nodes: Nodes, cluster_config: ClusterConfig = ClusterConfig
                     logging.info(f'--- TEARDOWN --- deleting created cluster {cluster.id}\n')
                     cluster.delete()
 
+    @staticmethod
+    def _set_up_proxy_server(cluster: Cluster, cluster_config, proxy_server):
+        proxy_name = "squid-" + cluster_name.suffix
+        proxy_server(name=proxy_name, port=consts.DEFAULT_PROXY_SERVER_IP, dir=proxy_name)
```
Comment (RazRegev) [inline discovery-infra/tests/base_test.py:121]: why DEFAULT_PROXY_SERVER_IP and not DEFAULT_PROXY_SERVER_PORT?

File: discovery-infra/test_infra/consts/consts.py [hunk 2]
```diff
@@ -48,6 +48,11 @@
 TEST_TARGET_INTERFACE = "vnet3"
 SUFFIX_LENGTH = 8
 
+DEFAULT_IPV6_SERVICE_CIDR = "2003:db8::/112"
```
Comment (YuviGold) [inline discovery-infra/test_infra/consts/consts.py:51]: Could it be handled dynamically as @RazRegev did?

File: discovery-infra/test_infra/consts/consts.py [hunk 3]
```diff
@@ -48,6 +48,11 @@
...
+DEFAULT_IPV6_CLUSTER_CIDR = "2002:db8::/53"
+DEFAULT_IPV6_HOST_PREFIX = 64
+DEFAULT_PROXY_SERVER_PORT = 3129
```
Comment (YuviGold) [inline discovery-infra/test_infra/consts/consts.py:54]: Would it work with 2 ipv6 tests in parallel?

File: discovery-infra/tests/conftest.py [hunk 4]
```diff
@@ -90,6 +90,13 @@ def get_kubeconfig_path():
     env_variables["vip_dhcp_allocation"] = False
 
 
+if env_variables["is_ipv6"]:
```
Comment (YuviGold) [inline discovery-infra/tests/conftest.py:93]: Should be part of the triggers dict after https://github.com/openshift/assisted-test-infra/pull/860 would be merged

**Review by YuviGold — APPROVED** [review]
LGTM

**Review by YuviGold — APPROVED** [review]

**Pull request merged**